js/snippetPreview.js

```diff
@@ -428,6 +428,11 @@ SnippetPreview.prototype.formatMeta = function() {
 	// Cut-off the meta description according to the maximum length
 	meta = meta.substring( 0, YoastSEO.analyzerConfig.maxMeta );
 
+	// Apply modification to the desc before showing it.
```

Comment:
Shouldn't we apply the modifications before we strip all tags and cut it off to the maximum length? The modification might add HTML or content we still want to strip and cut off.